src/types.js

```diff
@@ -20,17 +20,17 @@ export type PersonAbstract = {
   deathDate?: moment,
   influencedByCount: number,
   influencedCount: number,
-}
+|}
 
-export type PersonDetail = {
+export type PersonDetail = {|
   type: 'PersonDetail',
   uri: Uri,
   name: string,
   abstract?: string,
   birthPlace?: string,
-  birthDate?: moment,
-  deathDate?: moment,
+  birthDate: ?moment,
```

Comment:
Why the distinction between the question mark on the variable and the question mark on the data type?

Reply:
question mark on the property says "this might not be here." question mark on the type says "this needs to be _something_, but it might be a `moment` or a `null`."